lib/react_client.dart

```diff
@@ -551,14 +551,25 @@ SyntheticDataTransfer syntheticDataTransferFactory(events.SyntheticDataTransfer
     }
   }
   var effectAllowed;
+  var dropEffect;
+
   try {
     // Works around a bug in IE where dragging from outside the browser fails.
     // Trying to access this property throws the error "Unexpected call to method or property access.".
     effectAllowed = dt.effectAllowed;
   } catch (exception) {
     effectAllowed = 'uninitialized';
   }
-  return new SyntheticDataTransfer(dt.dropEffect, effectAllowed, files, types);
+
+  try {
+    // For certain types of drag events in IE (anything but ondragenter, ondragover, and ondrop), this fails.
+    // Trying to access this property throws the error "Unexpected call to method or property access.".
+    dropEffect = dt.dropEffect;
+  } catch (exception) {
+    dropEffect = 'uninitialized';
```

Comment:
"uninitialized" is a valid [`effectAllowed` value](https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer/effectAllowed#Values), but not a [`dropEffect` value](https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer/dropEffect#Values).

Should this be `'none'`?